docs/runbook/Features.md

```diff
@@ -1,145 +1,70 @@
-| NUMBER_OF_NODES_PER_SITE | int                                             | Maximum number of radio nodes allowed on each site. For CNs, this input will be ignored as it is always 1 |
+| NODE_CAPEX               | float                                           | Hardware cost of DN or CN                                                                                 |
+| NUMBER_OF_NODES_PER_SITE | int                                             | Maximum number of radio nodes allowed on each site; for CNs, this input must be 1 |
 | SECTOR_PARAMS            | a struct with the fields in the following table | The set of radio specification parameters                                                                 |
 
+The set of radio specification parameters are:
+
 | Sector Params Fields      |       |                                                                                                           |
 | ------------------------- | ----- | --------------------------------------------------------------------------------------------------------- |
 | HORIZONTAL_SCAN_RANGE     | float | Per-sector horizontal beamforming scan range of the antenna in degrees                                    |
 | NUMBER_SECTORS_PER_NODE   | float | Number of sectors in each node                                                                            |
 | ANTENNA_BORESIGHT_GAIN    | float | Antenna gain at boresight (dBi)                                                                           |
-| MAXIMUM_TX_POWER          | float | Maximum Transmission power in dBm                                                                         |
-| MINIMUM_TX_POWER          | float | Minimum Transmission power in dBm                                                                         |
+| MAXIMUM_TX_POWER          | float | Maximum transmission power in dBm                                                                         |
+| MINIMUM_TX_POWER          | float | Minimum transmission power in dBm                                                                         |
 | TX_DIVERSITY_GAIN         | float | Transmitter diversity gain in dB (e.g., polarization diversity)                                           |
 | RX_DIVERSITY_GAIN         | float | Receiver diversity gain in dB (e.g., polarization diversity)                                              |
 | TX_MISCELLANEOUS_LOSS     | float | Miscellaneous losses on the transmitter in dB (e.g., cable losses)                                        |
 | RX_MISCELLANEOUS_LOSS     | float | Miscellaneous losses on the receiver in dB (e.g., cable losses)                                           |
-| MINIMUM_MCS_LEVEL         | float | The minimum MCS level allowed                                                                             |
+| MINIMUM_MCS_LEVEL         | int   | The minimum MCS level allowed                                                                             |
```

Comment:
I asked in the chat some questions about this... based on that discussion, there may be an action item related to this.